.csscomb.json

```diff
@@ -0,0 +1,347 @@
+{
+  "always-semicolon": true,
+  "block-indent": 2,
+  "color-case": "lower",
+  "color-shorthand": true,
+  "eof-newline": true,
+  "leading-zero": false,
+  "quotes": "double",
+  "space-after-colon": 1,
+  "space-after-combinator": 1,
+  "space-after-selector-delimiter": 1,
+  "space-after-opening-brace": "\n",
+  "space-after-selector-delimiter": "\n",
+  "space-before-selector-delimiter": 0,
+  "space-before-colon": "",
+  "space-before-combinator": 1,
+  "space-before-opening-brace": 1,
+  "space-before-closing-brace": "\n",
+  "space-between-declarations": "\n ",
+  "vendor-prefix-align": true,
+  "verbose": true,
+  "strip-spaces": true,
+  "rules-delimiter": 1,
+  "sort-order": [
```

Comment:
Do we need this `sort-order` list in case we disable order check in scss-lint [bellow](https://github.com/fs/rails-base/pull/284/files#diff-6b5698395331b83eb1a6c1ecff9e6c0cR12)?